cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -801,7 +790,7 @@ define(["jquery", "edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers", "common/j
                         createMockSectionJSON({}, [mockSubsectionJSON])
                     ]);
                     createCourseOutlinePage(this, mockCourseJSON, false);
-                    outlinePage.$('.outline-subsection .configure-button').click();
+                    outlinePage.$('.outline-unit .configure-button').click();
```

Comment:
This is testing `does not show settings tab headers if there is only one tab to show`. As a result of the tab movement, subsections now always have 2 tabs (`Basic` and `Advanced`), so this test needs to be run on a unit, which only has the one tab (there's nothing on its `Basic` tab, so the visibility-editor-on-advanced gets collapsed down to the single untabbed view)